paddle/fluid/framework/details/reduce_op_handle.h

```diff
@@ -30,6 +30,32 @@
 namespace paddle {
 namespace framework {
 namespace details {
+struct CollectiveContext {
```

Comment:
define this under `platform/xxx_context` and initilize this in `ParallelExecutor`.

Reply:
Done.